### DISCUSSION TITLE
spacy in colab gets all-NaN slices

### DISCUSSION THREAD

**Discussion by diegorv1994** [explosion/spaCy#9726] [Help: Other Questions]
I am able to train a spacy model on google Colab on None mode, but when I try to use Colab's GPU I am not able to train the model. All slices are equal to Nan. Does anyone know how to fix it?
![captura_nlp](https://user-images.githubusercontent.com/56409940/142895578-a9e2c500-160a-4ff2-a43d-ba6720d9bc00.jpg)
y

**Comment by info2000** [comment]
did you installed the cuda113 package? and set prefer_gpu?
https://spacy.io/usage#gpu
also, take care not always colab enable a gpu

**Reply by diegorv1994** [reply]
Yes, I wrote require_gpu(). The output was True.

**Comment by polm** [comment]
I haven't seen that particular issue before, but take a look at [the FAQ](https://github.com/explosion/spaCy/discussions/8226) - colab seems to come with some special versions of libraries to support their hardware, and it seems most people are successful *not* installing cuda/cupy packages directly. 

Also please include the output of `spacy info` and other information about the packages in your environment, such as the output of `pip freeze`.
